pkg/pdatatest/pmetrictest/options.go

```diff
@@ -229,6 +230,56 @@ func orderDatapointAttributes(metrics pmetric.Metrics) {
 	}
 }
 
+// IgnoreDatapointsOrder is a CompareMetricsOption that ignores the order of datapoints.
+func IgnoreDatapointsOrder() CompareMetricsOption {
```

Comment:
Since this is a public package, we should include a changelog to announce this option.